src/Graviton/Rql/Visitor/MongoOdm.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Graviton\Rql\Visitor;
+
+use Graviton\Rql\AST\OperationInterface;
+use Doctrine\ODM\MongoDB\Query\Builder as QueryBuilder;
+
+class MongoOdm implements VisitorInterface
+{
+    /**
+     * @var QueryBuilder
+     */
+    private $queryBuilder;
+
+    /**
+     * @var string[]
+     */
+    private $queryOperations = array(
+        'and',
+        'or'
+    );
+
+    /**
+     * @var string[]
+     */
+    private $operationMap = array(
+        'eq' => 'equals',
+        'ne' => 'notEqual',
+        'lt' => 'lt',
+        'gt' => 'gt',
+        'lte' => 'lte',
+        'gte' => 'gte'
+);
```

Comment:
These do not use the Lexer::T_* constants as not to couple the visitor to the Lexer. I think the visitor should only depend on the AST. The "magic" strings here come from the ASTs ``getType()`` operation.

I should probably refactor the AST so it can express this as a composition of various classes. That would then lead to ``AST\Operation\Eq`` and so on. I'm not convinced that refactoring the AST will make the code more reusable though.